sched/Kconfig

```diff
@@ -184,6 +184,57 @@ config SYSTEM_TIME64
 		and/or if a very long "uptime" is required, then this option can be
 		selected to support a 64-bit wide timer.
 
+config ARCH_HAVE_ADJTIME
+	bool
+	default n
+
+config CLOCK_ADJTIME
```

Comment:
should we change to choice of CLOCK_ADJTIME and CLOCK_TIMEKEEPING? or remove CLOCK_TIMEKEEPING.

Reply:
I am not sure honestly. The current CLOCK_TIMEKEEPING implementation is weird and probably is not working properly. However I did not want to delete it in case of someone is actually using it and it is working fine for him. Hence two possible options. CLOCK_ADJTIME just enables adjtime and CLOCK_TIMEKEEPING timekeeping algorithms.